addons/binding/org.openhab.binding.harmonyhub/src/main/java/org/openhab/binding/harmonyhub/handler/HarmonyHubHandler.java

```diff
@@ -331,14 +332,16 @@ public HarmonyClient getClient() {
      * @param button
      */
     public void pressButton(int device, String button) {
-        buttonExecutor.execute(new Runnable() {
-            @Override
-            public void run() {
-                if (client != null) {
-                    client.pressButton(device, button);
+        if (buttonExecutor != null && !buttonExecutor.isShutdown()) {
+            buttonExecutor.execute(new Runnable() {
+                @Override
+                public void run() {
+                    if (client != null) {
+                        client.pressButton(device, button);
+                    }
                 }
-            }
-        });
+            });
+        }
```

Comment:
Sorry, I completely forgot about this!  I didn't bother as loosing a button command, which are mostly IR signals to control AV equipment, is not very critical and I don't personally think its that useful to log. I don't have a strong opinion however, so if someone else does I could add it in.